### PR TITLE
VP-5105: Add block for configuring order in product page

### PR DESCRIPTION
**Pull request by ilyawzrd**
customizeParts

### CODE REVIEW

**Comment by pushnitsa** [comment]
Too much free space :)
image

1. Image is missed
2. Markup is broken
image

**Comment by ilyawzrd** [comment]
Empty space is removed if tech specs for product are small
image

**Comment by ilyawzrd** [comment]
Fix card layout for configurable product
Screenshot_8

File: assets/js/product.js [hunk 1]
```diff
@@ -111,6 +111,19 @@ storefrontApp.controller('productController', ['$rootScope', '$scope', '$window'
             });
         }
 
+        $scope.changeGroupItem = function (productPart) {
+            var dialogInstance = dialogService.showDialog(productPart, 'changeConfigurationGroupItemDialogController', 'storefront.select-configuration-item-dialog.tpl');
+            dialogInstance.result.then(function (id) {
+                const foundIndex = $scope.productParts.findIndex(x => x.name == productPart.name);
```
Comment (pushnitsa) [inline assets/js/product.js:117]: Please use strict comparing

File: assets/js/product.js [hunk 2]
```diff
@@ -111,6 +111,19 @@ storefrontApp.controller('productController', ['$rootScope', '$scope', '$window'
...
+        $scope.changeGroupItem = function (productPart) {
+            var dialogInstance = dialogService.showDialog(productPart, 'changeConfigurationGroupItemDialogController', 'storefront.select-configuration-item-dialog.tpl');
+            dialogInstance.result.then(function (id) {
+                const foundIndex = $scope.productParts.findIndex(x => x.name == productPart.name);
+                $scope.productParts[foundIndex].selectedItemId = id;
+            });
+        };
+
+        $scope.getSelectedItem = function(configPart) {
+            const item = configPart.items.find(x => x.id == configPart.selectedItemId);
```
Comment (pushnitsa) [inline assets/js/product.js:123]: Use strict comparing

**Review by pushnitsa — CHANGES_REQUESTED** [review]

**Review by pushnitsa — APPROVED** [review]

**Pull request merged**